docs/src/main/asciidoc/_galleon/Galleon_layers.adoc

```diff
@@ -0,0 +1,863 @@
+[[wildfly_defined_galleon_layers]]
+=== WildFly Layers
```

Comment:
I mention elsewhere but should these be "WildFly Galleon Layers"?

Reply:
WildFly Layers